app/assets/stylesheets/new_styles/_base.scss

```diff
@@ -40,6 +42,15 @@ a { color : $link-blue  }
     height: 35px;
     width: 35px;
   }
+
+  &.medium {
+    height: auto;
+    width: auto;
+    max-width: 75px;
+  }
+
+  &.large {
+  }
```

Comment:
Oh, I guess that was a misunderstanding. I was just asking to remove the `.large` part but not the part about `.medium` which you added here because you removed it in single-post-view.scss.